torch/ao/quantization/fx/backend_config_dict/tensorrt.py

```diff
@@ -31,7 +31,9 @@ def get_tensorrt_backend_config_dict():
         "observation_type": ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT,
         "dtype_configs": [
             weighted_op_qint8_dtype_config,
-        ]
+        ],
```

Comment:
should we be adding some guidance for users on what this means along the way?

Reply:
Sure, I can add some comments for now? will have a more complete README later when these things are more stable